src/Tools/AI Test Toolkit/src/TestSuite/AITTestSuiteStatus.Enum.al

```diff
@@ -0,0 +1,42 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.TestTools.AITestToolkit;
+
+/// <summary>
+/// This enum has the Status of the AIT Test Suite.
+/// </summary>
+enum 149030 "AIT Test Suite Status"
+{
+    Extensible = false;
```

Comment:
Access = Public;

Reply:
No point in polluting, it is a default value